src/common/entity/can_toggle_state.ts

```diff
@@ -12,8 +13,7 @@ export default function canToggleState(
     return stateObj.state === "on" || stateObj.state === "off";
   }
   if (domain === "climate") {
-    // tslint:disable-next-line
-    return (stateObj.attributes.supported_features! & 4096) !== 0;
+    return supportsFeature(stateObj, 4096);
```

Comment:
4096 should be stored in a constant under `src/data/climate.ts`

Reply:
Yeah. Debated doing so but thought best to do later.